src/components/pool/MaxSlippageInput.tsx

```diff
@@ -10,7 +10,7 @@ const MESSAGE_TEXT_COLOR = 'rgba(204, 223, 237, 1)';
 
 const PREDEFINED_MAX_SLIPPAGE_OPTIONS = [
   {label: 'Low (5%)', value: '5.0'},
-  {label: 'Med (10%)', value: '10.0'},
```

Comment:
Are you sure you want to switch to 'Mid'? It is in the spec as 'Med'. I am not saying I don't support this change, I am just making sure you meant to do it!

Reply:
I don't have a strong preference, "Med" as an abbreviation for "Medium" just seemed odd to me. But now that I think about it I guess drinks + clothing work that way. I'll change it back